src/server/tiering/small_bins_test.cc

```diff
@@ -95,4 +100,18 @@ TEST(SmallBins, PartialStashDelete) {
   }
 }
 
+TEST_F(SmallBinsTest, Bug3240) {
```

Comment:
Maybe give it a proper name describing the scenarion and then keep a comment that is was a bug. I mean testing for fixed bugs is not worth it, but testing scenarios (not covered by tests) surely is.

Reply:
sure